bpmn-model/src/test/java/io/zeebe/model/bpmn/validation/ZeebeValidationTest.java

```diff
@@ -98,6 +98,61 @@
         singletonList(
             expect("flow", "Must only have payload mappings if its target is a parallel gateway"))
       },
+      {
+        Bpmn.createExecutableProcess("process")
```

Comment:
Maybe it makes sense to move the tests a separate file or add to `ZeebeMessageValidationTest` / `ZeebeTimerValidationTest`.

Reply:
I don't think it makes sense to put them in those files, since the validation here isn't specifically about the timer/message events (other than what is supported), but I can gladly create a new suite for boundary event tests. It's true that the more symbols we support, the bigger this file will get, and the more unwieldy.